From 0b859df9ff27d922078ef222244f7569fe1e6ce9 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 11 Aug 2020 15:57:00 +0300
Subject: [PATCH 37/39] mcm-imx8m-mini: config: Set CONFIG_IMX_SDMA=m for
 brcm,bcm4330-bt

Being configure as a part of the kernel causes the hci0 configuration issue.
uart does not work and all transactions get terminated with a tumeout.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/configs/mcm-imx8m-mini_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/mcm-imx8m-mini_defconfig b/arch/arm64/configs/mcm-imx8m-mini_defconfig
index 005c5de85f0f..5175fec11b39 100644
--- a/arch/arm64/configs/mcm-imx8m-mini_defconfig
+++ b/arch/arm64/configs/mcm-imx8m-mini_defconfig
@@ -654,7 +654,7 @@ CONFIG_DMADEVICES=y
 CONFIG_BCM_SBA_RAID=m
 CONFIG_FSL_EDMA=y
 CONFIG_FSL_EDMA_V3=y
-CONFIG_IMX_SDMA=y
+CONFIG_IMX_SDMA=m
 CONFIG_MV_XOR_V2=y
 CONFIG_MXS_DMA=y
 CONFIG_PL330_DMA=y
-- 
2.11.0

